marketplaceItems/questionApp/README.md

```diff
@@ -0,0 +1,14 @@
+# Question App
+
+This basic app allows you to signal that you would like to ask a question during meetings or events.
+
+Simply toggle the app on to put a question mark above your head. Admins in the domain can then see that you are waiting. The question mark will change color over time to show how long you've been waiting.
+
+When an admin is ready for you to ask your question, they will remove your question mark and you will hear a sound. your app will also toggle off at this time.
+
+## Known Issue
```

Comment:
you can remove this section if there's nothing in it :)

Reply:
I left that there in case we come across anything during testing. I'm going to wait until then and update.